etl-pipeline/processes/candidate_record_finder.py

```diff
@@ -3,7 +3,7 @@
 from typing import List, Set, Tuple, Optional, Any
 from sqlalchemy.exc import DataError
 from managers import DBManager, RedisManager
-
+import services.monitor as Monitor
```

Comment:
I'd import this `as monitor`, modules (and module aliases) are typically lower cased in python!